models/api.models.js

```diff
@@ -0,0 +1,11 @@
+const fs = require("fs").promises;
```

Comment:
If you would like to refactor to use fs promises you can require "fs/promises" here instead - would be better to do so rather that using a mix of callbacks and promises 😄